src/components/NavBar/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from 'react';
+import { Box, Button, Menu, MenuItem } from '@mui/material';
+import { NAVBAR } from 'utils/consts/navbar';
+import { INavbar } from 'utils/types/navbar.type';
```

Comment:
import { INavbar } from 'utils/types';

Reply:
the file extensions should be name.interface.ts for interfaces  and type for types